homeassistant/auth/__init__.py

```diff
@@ -519,6 +519,15 @@ def async_remove_refresh_token(self, refresh_token: models.RefreshToken) -> None
         for revoke_callback in callbacks:
             revoke_callback()
 
+    @callback
+    def async_edit_expiry_date(
+        self, refresh_token: models.RefreshToken, *, disable_expiry_date: bool
+    ) -> None:
```

Comment:
The name of the method suggests the expiry date can be edited, however the method allows disabling expiry